wrappers/android/tools/camera/src/main/java/com/intel/realsense/camera/SettingsActivity.java

```diff
@@ -340,7 +343,24 @@ public void onCheckedChanged(StreamProfileSelector holder) {
 
         Collections.sort(lines);
 
-        return lines.toArray(new StreamProfileSelector[lines.size()]);
+        return lines;
+    }
+
+    private void RemoveUnsupportedProfiles(List<StreamProfileSelector> streamProfiles){
+
+        // Entering Settings when all stream on (including confidence) result in an application crash.
```

Comment:
The problem is not when entering the Settings Activity, when when we leave it and go back to Preview Activity.
I would have changed the comment as:
// Confidence steam format is RAW8, and it is not supported for display.
// Its removal is necessary until format RAW8 display is enabled.
Feel free to use this or rephrase it by yourself.

Reply:
I'd move this comment to right before line 356, too

Reply:
Or even 362

Reply:
I did it,

but when someone will read this comment 1 year from now and say hey, I can add support for raw8 and remove this patch, he will not know about the problem, he don't have a ticket to read, and no information on the comment.
And he will 99% won't test the flow that caused the  issue.

Don't you guys think it's necessary?

Reply:
I think the ticket number is harmless -- but you're referring to the other comment, right?